1-exercises/A-undefined/exercise.js

```diff
@@ -12,25 +12,28 @@
 // Example 1
 let a;
 console.log(a);
```

Comment:
Hi Temwa. The default "value" that is given to a javascript variable like "a" is undefined. This is when the code does not show that the variable has been explicitly assigned a value (e.g. let a = "Hello";). I guess we can call "undefined" a special value. But good job either way.